### DISCUSSION TITLE
srsUE not connecting with USRP N210

### DISCUSSION THREAD

**Discussion by sruthi81** [srsran/srsRAN_4G#1409] [Q&A] [answered]
While I am connecting
![WhatsApp Image 2024-10-24 at 10 34 21](https://github.com/user-attachments/assets/525301bf-73a2-4adf-9589-d57f88c05e99)
 the N210 USRP with srsUE, then getting the sampling rate error.Kindly help me to resolve the issue.

**Comment by andrepuschmann** [comment, marked as answer]
The USRP N210 is not supported by srsRAN
